docs/config/01-configuration-file.md

```diff
@@ -280,6 +280,34 @@ httpsServerOptions: {
 **Description:** A list of log appenders to be used. See the documentation for [log4js] for more information.
 
 
+## middleware
+**Type:** Array
+
+**Default:** `[]`
+
+**Description:** List of names of additional middleware you want the karma server to use.
+
+You must have installed the middleware via a plugin/framework (either inline or via NPM). An example of custom inline middleware is shown below. Additional information can be found in [plugins].
```

Comment:
Also it would be good to note that the middleware is used in the order they are included, as the order can matter if I remember correctly.